.github/workflows/push-asset-readme-update.yml

```diff
@@ -2,15 +2,23 @@ name: Plugin asset/readme update
 on:
   push:
     branches:
-    - trunk
+      - trunk
 jobs:
   trunk:
     name: Push to trunk
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-    - name: WordPress.org plugin asset/readme update
-      uses: 10up/action-wordpress-plugin-asset-update@stable
-      env:
-        SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
-        SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+      - uses: actions/checkout@v2
+      - name: install node v10
```

Comment:
Minor thing but this says we're installing node v10 but the lines below mention v12